kima/assetlist.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "kima",
+  "assets": [
+    {
+      "description": "The native token of Kima Chain",
+      "denom_units": [
+        {
+          "denom": "uKIMA",
+          "exponent": 0
+        },
+        {
+          "denom": "KIMA",
+          "exponent": 6
+        }
+      ],
+      "base": "uKIMA",
+      "name": "KIMA",
+      "display": "KIMA",
+      "symbol": "KIMA",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kima/images/kima.png",
```

Comment:
The images for the equivalent testnet asset should be removed from the kimatestnet directory. We only need one copy of these images